<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultationResult</class>
 <widget class="QDialog" name="ConsultationResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>395</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>-12821</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Consultation - Results</string>
  </property>
  <widget class="QListWidget" name="FactList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>170</width>
     <height>270</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>270</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>270</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="FactValueLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fact Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FactValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FactListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Facts:</string>
   </property>
  </widget>
  <widget class="Line" name="Divider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>20</width>
     <height>375</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>375</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>375</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTreeWidget" name="ProcessTree">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>25</y>
     <width>340</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>340</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>340</width>
     <height>360</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>110</number>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>110</number>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>What</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Confidence</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="ProcessTreeLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Consultation Process:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfidenceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>345</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confidence Factor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Confidence">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>365</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FactList</tabstop>
  <tabstop>ProcessTree</tabstop>
  <tabstop>FactValue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>FactList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ConsultationResult</receiver>
   <slot>FactSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FactSelected()</slot>
 </slots>
</ui>
